src/app/linear-trend.component.html

```diff
@@ -0,0 +1,45 @@
+<mat-drawer-container
+  class="lineartrend-container"
+  style="width:150vh; margin-bottom: 0px; overflow:hidden; z-index: 0"
+>
+  <div class="title_text" class="col s12" align="right">
+    <button
+      mat-icon-button
+      (click)="this.closeLinearTrend()"
+      class="closeBtn"
+      matTooltip="Close Linear Trend Chart"
+    >
+      <mat-icon>close</mat-icon>
+    </button>
+  </div>
+  <mat-drawer class="lt-sidenav" mode="side" opened="true">
+    <mat-list>
+      <p>Select</p>
```

Comment:
"Select" on its own looks kind of weird to me in the linear trend menu.  Maybe "Select Month and Level" or "Linear Trend" or maybe tool tips on the drop down buttons instead? Just a thought.